CompetitionTemplate/src/competition/subsystems/arm/ArmTargetSubsystem.java

```diff
@@ -0,0 +1,25 @@
+package competition.subsystems.arm;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+
+@Singleton
+public class ArmTargetSubsystem extends BaseSubsystem{
+    double targetAngle;
+    
+    @Inject
+    public ArmTargetSubsystem (){
+        
+    }
+    
+    public double setTargetAngle(double targetAngle){
+        this.targetAngle = targetAngle;
```

Comment:
It would be nice if this was a DoubleProperty so we could see what it was trying to go to on the dashboard